src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2229,11 +2228,12 @@ public T block(Duration timeout) {
 	 * sequence
 	 *
 	 * @return a new {@link Mono}
+	 * @deprecated Use {@link #otherwise(Predicate, Function)} instead.
```

Comment:
should be `#switchOnError` not `#otherwise`. Maybe also indicate it will be removed in 3.1.0.RELEASE (or any pre-release after M1)?